modules/ROOT/pages/deploying-receive-endpoints.adoc

```diff
@@ -47,7 +46,7 @@ With this configuration, the external AS2 endpoint URL is `+https://mythical.lb.
 
 The external URL for an HTTPS endpoint is `+https://{DLB-domain}/{Input-path}/{base-path}/{message-type}+`.
 
-For example, if the name of your DLB domain is `mythical.lb.anypointdns.net`, the base path you configured for the endpoint is `send2`, and the runtime application name is `b2b-inbound-http-ognq`, then your administrator can add a URL mapping rule in the DLB settings to forward incoming requests to the runtime application within your VPC. For example:
+For example, if the name of your DLB domain is `mythical.lb.anypointdns.net`, the base path you configured for the endpoint is `send2`, and the runtime application name is `b2b-inbound-http-ognq`, then your administrator can add a URL mapping rule in the DLB settings to forward incoming requests to the runtime application within your Anypoint VPC. For example:
 
 image::url-mapping-rules-https.png[URL mapping rules for HTTPS in the DLB settings]
 
```

Comment:
@dhirsch1 - The section "External URL and Mapping Rules for an HTTP Receive Endpoint" should be removed. All HTTP receive endpoint in CloudHub is by default HTTPS only.